CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-vue-wallet ChangeLog
 
+## 28.1.1 - 2024-04-dd
```

Comment:
Generally, if `### Added` appears (a new feature has been added), it needs to be minimally a minor release. Patch releases are only for fixes (`### Fixed`).

```suggestion
## 28.2.0 - 2024-04-dd
```

Reply:
Thanks for the clarifications!